articles/data-factory/parameters-data-flow.md

```diff
@@ -106,6 +106,9 @@ For example, if you wanted to map a string column based upon a parameter `column
 
 :::image type="content" source="media/data-flow/parameterize-column-name.png" alt-text="Passing in a column name as a parameter":::
 
+> [!NOTE]
+> In dataflow, string interpolations are not supported and one needs to concatenate the dynamic expression into a string value for dynamic logic purpose.
```

Comment:
Would this example help?
```suggestion
> [!NOTE]
> In data flow expressions, string interpolation (substituting variables inside of the string) is not supported. Instead, concatenate the expression into string values. For example, `'string part 1' + $variable + 'string part 2'`
```

Reply:
@JasonWHowell Thank you so much for your help in enhancing it further with an example. The example looks really helpful. 

Appreciate much!